partner_credit_limit/models/partner.py

```diff
@@ -92,3 +95,23 @@ def get_allowed_sale(self):
         for partner in self:
             partner.allowed_sale = not partner.credit_overloaded and \
                 not partner.overdue_credit
+
+    def _check_limit_credit(self):
```

Comment:
`_check_credit_limit`

Limit credit is semantically incorrect. Yes, I know, it was that way.

Reply:
Done